extensions/servlet/test/com/google/inject/servlet/ServletDefinitionPathsTest.java

```diff
@@ -93,6 +93,12 @@ protected void service(HttpServletRequest servletRequest,
     expect(request.getServletPath())
         .andReturn(requestPath);
 
+    // computePath()
```

Comment:
this line is completely new. the below code resembles the newly introduced logic in computePath() method.